content/en/docs/Installation/raspberry.md

```diff
@@ -32,7 +32,7 @@ Extract the `img` file from a container image as described [in this page]({{< re
 Plug the SD card to your system. To flash the image, you can either use Etcher or `dd`. Note it's compressed with "XZ", so we need to decompress it first:
 
 ```bash {class="only-flavors=openSUSE+Leap-15.6,openSUSE+Tumbleweed,Ubuntu+20.04,Ubuntu+22.04,Alpine+3.19"}
-xzcat {{<image variant="standard" model="rpi4" arch="arm64" suffix=".img.xz">}} | sudo dd of=<device> oflag=sync status=progress bs=10MB
+sudo dd if={{<image variant="standard" model="rpi4" arch="arm64" suffix=".img.xz">}} of=<device> oflag=sync status=progress bs=10MB
```

Comment:
if the suffix is `.img.xz` then `xzcat` is still needed.

Reply:
ah crap, this is wrong, is not the proper suffix, good catch!